### DISCUSSION TITLE
Leveraging marketing strategies to eclipse WordPress competitors

### DISCUSSION THREAD

**Discussion by jennimckinnon** [WordPress/Marketing-Team#202] [General]
## Overview: TL;DR

Launching a series of strategic marketing strategies and campaigns will assist in eclipsing [WordPress.org](https://wordpress.org) competitors who seek to poach the internally determined influential WordPress experts they incentivized to switch to, and promote their platforms and services.

## Concept and history

In the [Marketing channel](https://wordpress.slack.com/archives/C0GKJ7TFA/p1677760310729459) on [Slack](https://slack.com/), [@jonoaldersonwp](https://profiles.wordpress.org/jonoaldersonwp/) reported that some of the biggest influencers in the SEO space are continuing to abandon their self-hosted WordPress sites because [Wix](https://www.wix.com/) is incentivizing them in various ways.

Both [Barry Adams](https://twitter.com/badams/status/1631269875082575874) and [Aleyda Solis](https://twitter.com/aleyda/status/1592811859257745408) tweeted that they redesigned their site with Wix in a promotional way.

[@jonoaldersonwp](https://profiles.wordpress.org/jonoaldersonwp/) explained that Wix is spending time and resources on assisting them, and incentivizing the move with free support, promotion, discounted or free services, and more.

Moreover, this is occurring intentionally as a part of a marketing strategy to leverage the experts’ influence to promote Wix at the expense of other platforms – like WordPress – that are no longer able to do the same.

An advisory council of industry influencers (who are under heavy NDA and non-compete agreements) are in place that also includes Barry Adams as well as Aleyda Solis.

A comprehensive marketing strategy is needed to bring WordPress back to the forefront of the public image as the robust, agreeable, and infinitely customizable platform that it is today, and in the future.

## Objective

The Make WordPress marketing team would like to work on an in-depth strategy that gracefully counteracts and eclipses Wix’s strategy as well as other competitors that may be also applying similar approaches.

## Proposed strategies and ideas

[@santanainniss](https://profiles.wordpress.org/santanainniss/) mentioned leveraging influencer marketing and [Five for the Future](https://wordpress.org/five-for-the-future/) would be a good option to assist in implementing our marketing strategy.

[@ninianepress](https://profiles.wordpress.org/ninianepress/) (me) agreed and added:

- A deeper strategy will be required.
- WordPress’ 20th Birthday is coming up and can be leveraged.
- Smaller campaigns can additionally be created as feeders to the larger [Campaign #199](https://github.com/WordPress/Marketing-Team/discussions/199).
- These details should be used to inform Make WordPress’ marketing strategies for the next 3+ years.
- An opinionated “certified experts” page could be created similar to the [WordPress Hosting page on WordPress.org](https://wordpress.org/hosting/) that recommends agencies, consultants, developers, marketers, and more that can supplement the [WordPress Support Forum](https://wordpress.org/support/forums/).

In addition to supplementing the last point above, [@jonoaldersonwp](https://profiles.wordpress.org/jonoaldersonwp/) also mentioned that a directory of individuals, organizations, agencies, and companies that participate in [Five for the Future](https://wordpress.org/five-for-the-future/) could be created.

[@ninianepress](https://profiles.wordpress.org/ninianepress/) added that there already is a [Pledges page](https://wordpress.org/five-for-the-future/pledges/) and a [proposal](https://make.wordpress.org/updates/2018/11/05/proposal-five-for-the-future-acknowledgement-page/) for this concept, but there’s a larger opportunity for creating a more robust and publicized version of the Pledges page to further promote and incentivize WordPress.

_For the full discussion, please see the [Slack transcript](https://wordpress.slack.com/archives/C0GKJ7TFA/p1677760310729459) as well as the [supplemental transcript](https://wordpress.slack.com/archives/C0GKJ7TFA/p1677760819445649)_.

## Opportunities and challenges

[@santanainniss](https://profiles.wordpress.org/santanainniss/) mentioned Wix can build a website, but it’s a walled garden that’s anti-ethical to open source.

[@jonoaldersonwp](https://profiles.wordpress.org/jonoaldersonwp/) pointed out that the public image of Wix is that it’s easier to use, sleeker, easier to publish and manage content as well as easier to market.

Comparatively, WordPress is cumbersome, clunky, stuck in a rut between the classic, TinyMCE editor, Gutenberg, and the Full Site Editor (FSE). Users have to know what they want or need, then go searching for it, and the theme and plugin repositories are nightmarish in the sense that there’s no curation, and no discovery mechanisms.

There's also no help, the support forums are impenetrably unwelcoming, and folks just want things to be easy.

Over 60,000 plugins also isn’t what people want. They want it to be effortless to get set up and write their content, and sell their products or services, but WordPress doesn’t facilitate that.

Key themes [@jonoaldersonwp](https://profiles.wordpress.org/jonoaldersonwp/) hears from users that drive the migration are:

- Ease of onboarding
- Availability of templates and layouts designed for specific scenarios, without the headache of theme or plugin discovery
- Bundled and core functionality, without the headache of plugin discovery
- Stability, support, engagement

[@ninianepress](https://profiles.wordpress.org/ninianepress/) suggested that a possible idea to address the headache of finding plugins and discovery is to have dedicated people in support (especially with Five for the Future) to take the time to understand a user’s need through a ticket or similar, and then they advise on a plugins list.

[@jonoaldersonwp](https://profiles.wordpress.org/jonoaldersonwp/) pointed out that by the time we get to assisting individual accounts with plugin discovery, it's too late. We need top of funnel messaging (landing pages, onboarding flows), and curated and opinionated solutions.

Otherwise users won’t reach the point where we can engage with them; they'll already be on Wix.

[@ninianepress](https://profiles.wordpress.org/ninianepress/) agreed and suggested a multi-faceted approach that includes short, medium, and long-term planning.

### Learn courses in the WordPress dashboard

In a separate, asynchronous conversation, [@leogopal](https://profiles.wordpress.org/leogopal/) and [@ninianepress](https://profiles.wordpress.org/ninianepress/) discussed further possible opportunities for improving the user experience:

Wix helps with education and onboarding which gives the impression that it’s easier to use than a self-hosted [WordPress.org](http://wordpress.org/) website for new users.

Education in WordPress is actively being worked on, and it’s already available through [learn.wordpress.org](http://learn.wordpress.org/). But, it’s not where the user is – they’re in their WordPress admin dashboard.

A possibility that could work, would be contextual help that's available in different areas of WordPress to include courses and training that’s relevant as well as meets the user where they are both in skill level, and on-page.

Another powerful approach could be much like our Meetup or Events widgets in the admin dashboard but for presenting learn training with the ability for users to search for what they want to learn and have the opportunity to do so right in their dashboard.

Similarly, WordPress.tv content could potentially be displayed, including in search results.

Since there are already Meetup and Events widgets in place, adding an additional dashboard widget is a low-cost venture.

## Support from other Make Teams

This campaign will need additional support from other [Make WordPress teams](https://make.wordpress.org/) including, but necessarily limited to:

- Training
- Support
- Documentation
- Core
- Design
- Meta
- Community
- Plugins
- Themes

## Other notes and considerations

Please comment below with feedback, ideas, and anything else you feel is important so that we can discuss this matter further, and get started.

“WordPress cannot do everything, but WordPress, with help, can do everything.”
—[Leo Gopal](https://poststatus.com/member-spotlight-leo-gopal/), Community Rep 

We’ve got this – let’s press forward.

**Comment by Lennair19** [comment]
Hello, this is a compelling topic. Thank you for all these details. I want to add some suggestions regarding plugins. 

You state: "Over 60,000 plugins also is not what people want. They want it to be effortless to get set up, write their content, and sell their products or services, but WordPress does not facilitate it."

Here are some suggestions to facilitate the use of plugins: 
1- Eliminate plugins that are no longer updated and, as a result, are incompatible with the current WordPress version. There are abandoned plugins that create confusion. Normally, before installing the plugin, you can see if it is compatible with the current version of WordPress. However, new people may not realize this. Once they install the plugin, the website may not work due to incompatibilities with other plugins or when WordPress is updated. Finding the plugin that causes problems could be a nightmare. Some companies are abandoning WordPress and going to a static HTML and CSS web to avoid this. 

2-Encorage developers to eliminate plugins that they won't update anymore. 

3-Automaticly remove plugins that are out of date for some time. 

4- Add filters to the plugin repository such as areas of expertise and functionalities: for example, e-commerce, payment, SEO, contact forms, CRM, fonts, image galleries, security, etc. 

5- Add a list and video of recommended plugins based on functionalities in the area of plugins the first time users access it after installing WordPress.
 
These are some ideas to organize the plugins to facilitate their use. 
I will love to hear your feedback. Thanks.

**Comment by jennimckinnon** [comment]
@Lennair19 Thanks for your ideas. I definitely understand where you're coming from and it's definitely frustrating to see plugins that aren't actively being developed, especially for end users.

As I understand it, the reason why these types of plugins aren't just deleted is because they exist due to the GPL license; that is to say, WordPress uses the GPL license where you're able to freely use and modify the software under this license as long as iterations  are made freely available under the same license; so, the plugins are there for others to modify and release new versions that can be actively developed.

There has been a successful effort to re-label them for clarity.

In terms of this topic discussion, your proposed changes are outside the scope of what the Make WordPress Marketing Team can do. I think this idea is better suited to discuss with the Meta Team if it's something you feel passionately about.

**Comment by jimthemarketingmindz-glitch** [comment]
I agree that competing solely on features is difficult when platforms like Wix invest heavily in onboarding, support, and influencer partnerships. The bigger opportunity for WordPress is to improve the user journey from day one. 

Many users don't leave because WordPress lacks power; they leave because finding the right themes, plugins, and guidance feels overwhelming.

I've seen a similar approach work in digital marketing, where businesses focus on education, curated recommendations, and clear onboarding instead of simply promoting features. 
Companies like [themarketingwizardz.com](themarketingwizardz.com) often emphasize helping users understand the right solution before asking them to make decisions. 

WordPress could benefit from a similar strategy by creating guided experiences, trusted recommendations, and stronger educational pathways that make adoption feel effortless rather than complicated.
